test/all-options.test.js

```diff
@@ -33,6 +27,14 @@ describe('when applied with all options', () => {
           return `License information can be found in ${licenseFile}`;
         },
       },
+      uglifyOptions: {
```

Comment:
`warningFilters` : I think it can be used by some people to filter for example on node_modules and still see warnings about their code, maybe to check dead code presence, or optimisation.

`extractComment`: It is for copyright/license reason I think => that way, the file you load is steel small, plus a preamble redirecting to another file containing all the copyrights/licenses.